requirements.txt

```diff
@@ -1 +1 @@
-in-toto==0.1.1
\ No newline at end of file
+in-toto==0.2.3
```

Comment:
I thought about a `>=` here, but since there's no automated guarantees that the code will work, I think it's better to pin this version for now.

Reply:
Pinning the exact version is fine. 

Originally this was part of the main repo, but we removed it to keep the reference implementation repo cleaner (see https://github.com/in-toto/in-toto/pull/127).